department-of-veterans-affairs/va.gov-cms
# VACMS-000: Remove .git sub-directories after composer install & update

## Description

This fixes that really annoying issue where `composer install` prompts you if you want to overwrite modified changes to certain dependencies/modules. We don't need the .git directories, unless we are working on an upstream module, and in that case we will likely clone it outside of the repo and symlink as needed.

## Testing done
Ran this locally with `composer install`. 

## Screenshots
![image](https://user-images.githubusercontent.com/1504756/206591598-dec39d08-12a2-42d8-817d-46ddc1110b86.png)


## QA steps
1. `cd docroot/modules/cer`
2. Verify a `git status` returns git info about the `cer` module, not the CMS repo
3. Run `composer install`
4. Verify it finishes with a step that says "Removing .git sub-directories...`
5. Run `git status` again in the `cer` directory, you should see info about this branch instead

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
